R/regression.R

```diff
@@ -347,13 +347,16 @@ Regression <- function(formula = NULL,
     # Check if stackable data is input
     if (stacked.data.check)
     {
-        stacked.data.output <- processAndStackData(unstacked.data, formula, interaction, subset, weights)
+        stacked.data.output <- processAndStackData(unstacked.data, formula, interaction, subset, weights, missing)
         # Update relevant terms
         data <- stacked.data.output[["data"]]
         formula <- input.formula <- stacked.data.output[["formula"]]
         interaction <- stacked.data.output[["interaction"]]
         subset <- stacked.data.output[["subset"]]
         weights <- stacked.data.output[["weights"]]
+        n.orig.stacked.cases <- stacked.data.output[["n.orig.stacked.cases"]]
+        n.stacked.cases.removed <- stacked.data.output[["n.stacked.cases.removed"]]
+        remove("stacked.data.output")
```

Comment:
If I recall correctly you typically don't need to `remove` objects in R if they are not returned in a function since they should be removed anyway and not take up space when the code is out of scope. Unless you observed `R` was running out of memory later in this function in an intermediate step?